inactive_session_timeout/models/res_users.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE - Cédric Pigeon
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import http, models, registry, SUPERUSER_ID
+from odoo.api import Environment
+from odoo.http import request, root
+
+from os import utime
+from os.path import getmtime
+from time import time
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @classmethod
+    def _check_session_validity(cls, db, uid, passwd):
+        if not request:
+            return
+        session = request.session
+        session_store = root.session_store
+        with registry(db).cursor() as cr:
+            env = Environment(cr, SUPERUSER_ID, {})
+            param_obj = env['ir.config_parameter']
+            delay, urls = param_obj.get_session_parameters(db)
```

Comment:
Again, you'll have to change this for 2 calls to `ir.config_parameter.get_param` if you delete `ir_config_parameter.py` file as suggested above.